### PR TITLE
Added possibility to disable automatic strings to unicode translation

### PR DESCRIPTION
**Pull request by IgorSasovets**
## Proposed changes

This is a fix for wdio automatic translation to unicode  [issue](https://github.com/webdriverio/webdriverio/issues/4936). In general, user can't send keys that are predefined in the UNICODE_CHARACTERS [list](https://github.com/webdriverio/webdriverio/blob/master/packages/webdriverio/src/constants.js).

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [x ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [x] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [ ] I have added proper type definitions for new commands (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by IgorSasovets** [comment]
UPDATE: I will revert my changes in package.json & package-lock.json files

File: packages/webdriverio/src/utils.js [hunk 1]
```diff
@@ -82,7 +82,7 @@ export function getBrowserObject (elem) {
 /**
  * transform whatever value is into an array of char strings
  */
-export function transformToCharString (value) {
+export function transformToCharString (value, { translateToUnicode } = { }) {
```
Comment (christian-bromann) [inline packages/webdriverio/src/utils.js]: given this is an internal utility I would advocate for this structure:
```suggestion
export function transformToCharString (value, translateToUnicode) {
```

File: packages/webdriverio/tests/commands/element/setValue.test.js [hunk 2]
```diff
@@ -44,3 +44,29 @@ describe('setValue', () => {
         expect(request.mock.calls[3][0].body.text).toEqual('12true[1,2]')
     })
 })
+
+describe('should allow to add value to an input element as workaround for /webdriverio/issues/4936', () => {
```
Comment (christian-bromann) [inline packages/webdriverio/tests/commands/element/setValue.test.js]: Can we merge this into the `describe` block above?

File: packages/webdriverio/tests/utils.test.js [hunk 3]
```diff
@@ -90,6 +90,17 @@ describe('utils', () => {
...
+
+        it('should transform string without converting to unicode', () => {
+            expect(transformToCharString('Delete', { translateToUnicode: false })).toEqual(
+                ['D', 'e', 'l', 'e', 't', 'e'])
+            expect(transformToCharString('Back space', { translateToUnicode: false })).toEqual(
+                ['B', 'a', 'c', 'k', ' ', 's', 'p', 'a', 'c', 'e'])
+            expect(transformToCharString('Backspace', { translateToUnicode: false })).toEqual(
+                ['B', 'a', 'c', 'k', 's', 'p', 'a', 'c', 'e'])
+            expect(transformToCharString('Pageup', { translateToUnicode: false })).toEqual(
+                ['P', 'a', 'g', 'e', 'u', 'p'])
```
Comment (christian-bromann) [inline packages/webdriverio/tests/utils.test.js]: Can we add cases where `translateToUnicode` is set to `true`?

File: packages/webdriverio/src/utils.js (hunk 1 above)
Comment (IgorSasovets) [reply packages/webdriverio/src/utils.js]: I agree with you. Should we define the default value for the new property as I did ?

File: packages/webdriverio/tests/commands/element/setValue.test.js (hunk 2 above)
Comment (IgorSasovets) [reply packages/webdriverio/tests/commands/element/setValue.test.js]: Yes, I'll do it

File: packages/webdriverio/tests/utils.test.js (hunk 3 above)
Comment (IgorSasovets) [reply packages/webdriverio/tests/utils.test.js]: Yes, no problem

File: packages/webdriverio/src/utils.js (hunk 1 above)
Comment (christian-bromann) [reply packages/webdriverio/src/utils.js]: default property should mimic current behavior so we don't break user tests.

Comment (IgorSasovets) [reply packages/webdriverio/src/utils.js]: Thanks, I'll update it

**Comment by christian-bromann** [comment]
@IgorSasovets also given that this will not land in v6 (which is the current master branch) would you mind changing the target branch to `v5`?

**Comment by IgorSasovets** [comment]
@christian-bromann , done. Should I create a new branch from the `v5`, add required changes and then open a new PR? Otherwise, there will be some conflicts and a lot of additional commits will be merged into the `v5`.

**Comment by christian-bromann** [comment]
> Should I create a new branch from the `v5`, add required changes and then open a new PR?

Yeah I think that would be the best.

**Comment by IgorSasovets** [comment]
Ok, closing this PR.

**Comment by IgorSasovets** [comment]
@christian-bromann , please check out my new [pr](https://github.com/webdriverio/webdriverio/pull/4946)

**Pull request closed without merging**